Find a functionally equivalent code:android.view.GraphicBuffer.create:COMMENT
Method Modifier: public      static      
Comment:/**
 * Creates new <code>GraphicBuffer</code> instance. This method will return null
 * if the buffer cannot be created.
 *
 * @param width The width in pixels of the buffer
 * @param height The height in pixels of the buffer
 * @param format The format of each pixel as specified in {@link PixelFormat}
 * @param usage Hint indicating how the buffer will be used
 *
 * @return A <code>GraphicBuffer</code> instance or null
 */

@@ -1,5 +1,5 @@
 {
-    int nativeObject = nCreateGraphicBuffer(width, height, format, usage);
+    long nativeObject = nCreateGraphicBuffer(width, height, format, usage);
     if (nativeObject != 0) {
         return new GraphicBuffer(width, height, format, usage, nativeObject);
     }

